recipe/conda_build_config.yaml

```diff
@@ -462,7 +462,7 @@ libprotobuf:
 librdkafka:
   - 0.11.5
 librsvg:
-  - 2.44.3
+  - 2
```

Comment:
I guess this should be `2.44.14` to ensure we are getting the latest one with `run_exports` (after we merge https://github.com/conda-forge/librsvg-feedstock/pull/26 and remove the old build without `run_exports`.)

Reply:
We'll have to keep bumping the version if it's `2.44.13`.

Reply:
Not sure what you mean? Bump the version of what depend on it? If so wouldn't a migration be enough to take care of this problem?

Reply:
We don't need a migration until version 3.0.
Say we bump `pango` which is a dependency of librsvg and `pango` was bumped in `librsvg==2.44.14` (a future version). Then we need to bump `librsvg` here to `2.44.14` as well which is unnecessary work IMO and if we had left this as `2` we don't need to do that.